@kiva/kv-components/vue/KvProgressBar.vue

```diff
@@ -59,28 +66,35 @@ export default {
 			required: true,
 		},
 	},
-	data() {
-		return {
-			loaded: false,
-		};
-	},
-	computed: {
-		percent() {
-			const percent = ((this.value - this.min) / (this.max - this.min)) * 100;
-			const rounded = Math.round(percent * 10) / 10; // Keep percents to 1 demical places (12.3%)
+	setup(props) {
+		const {
+			min,
+			max,
+			value,
+		} = toRefs(props);
+		const loaded = ref(false);
+
+		const percent = computed(() => {
+			const percentValue = ((value.value - min.value) / (max.value - min.value)) * 100;
+			const rounded = Math.round(percentValue * 10) / 10; // Keep percents to 1 decimal places 12.3%
 			const clamped = Math.min(Math.max(rounded, 0), 100); // Always between 0 and 100%
 			return clamped;
-		},
-	},
-	mounted() {
-		this.$nextTick(() => {
-			this.animateProgressBar();
 		});
-	},
-	methods: {
-		animateProgressBar() {
-			this.loaded = true;
-		},
+
+		const animateProgressBar = () => {
+			loaded.value = true;
+		};
+
+		onMounted(() => {
+			nextTick();
+			animateProgressBar();
```

Comment:
This animation isn't happening on load. I think nextTick is async, so should be `await nextTick();`

Reply:
Fixed, thank!